### PR TITLE
Adding token: SAIL from chain OSMOSIS

### PR DESCRIPTION
**Pull request by nick134-bit**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [x] Ready -- A liquidity pool has been created. The pool ID is: 1462
- [ ] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2593,6 +2593,12 @@
       "base_denom": "factory/osmo1279xudevmf5cw83vkhglct7jededp86k90k2le/RAPTR",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo1rckme96ptawr4zwexxj5g5gej9s2dmud8r2t9j0k0prn5mch5g4snzzwjv/sail",
+      "twitter_URL": "https://twitter.com/Sail_DAO_",
+      "osmosis_verified": true
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2601]: please also add osmosis_unlisted so we can first validate that everything looks OK.

**Comment by JeremyParish69** [comment]
Will wait for the Pool ID before merging.

**Comment by JeremyParish69** [comment]
The chain-registry is showing as a conflict, could you please resolve that?

**Comment by nick134-bit** [comment]
@JeremyParish69 just did

**Pull request closed without merging**